projects/pgai/pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
     "datadog_lambda>=6.9,<7.0",
     "pytimeparse>=1.1,<2.0",
     "voyageai>=0.3.1,<0.4.0",
+    "litellm>=1.57.5,<1.58.0",
```

Comment:
any errors in 1.58? or just cautious?

Reply:
Actually I think when I started working on this 1.58 wasn't out yet. Will update.